eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/consumer/LiteConsumer.java

```diff
@@ -234,7 +237,13 @@ public void run() {
     }
 
     public boolean unsubscribe(List<String> topicList, String url) throws Exception {
-        subscription.removeAll(topicList);
+        Set<String> unSub = new HashSet<>(topicList);
+        for (SubscriptionItem item:subscription) {
+            if (unSub.contains(item.getTopic())) {
+                subscription.remove(item);
```

Comment:
Here, may cause ConcurrentModification Exception, use iterator may be better, please take a look